website/content/en/preview/tasks/deprovisioning.md

```diff
@@ -146,6 +146,7 @@ If you want to terminate a node with a `do-not-evict` pod, you can simply remove
 
 Consolidation will be unable to consolidate a node if, as a result of its scheduling simulation, it determines that the pods on a node cannot run on other nodes due to inter-pod affinity/anti-affinity, topology spread constraints, or some other scheduling restriction that couldn't be fulfilled.
 
-### Controllerless Pods (Consolidation Only)
+{{% alert title="Note" color="warning" %}}
+Starting with Karpenter v0.19.0, pods without an ownerRef (also called "controllerless" or "naked" pods) will now be evicted by default during node termination and consolidation.  Users can prevent controllerless pods from being voluntarily disrupted by applying the `karpenter.sh/do-not-evict: true` annotation to the pods in question.
```

Comment:
Let's avoid leaking historical context in the logs. That's what doc versioning is for.

Does it make sense to include this warning in `### Scheduling Constraints (Consolidation Only)`?

Also worth talking about the larger behavior, where it blocks the whole node, not just the pod.
```suggestion
Pods without an ownerRef (also called "controllerless" or "naked" pods) will be evicted during voluntary node disruption, such as expiration or consolidation. A pod with the annotation `karpenter.sh/do-not-evict: true` will cause its node to be opted out from voluntary node disruption workflows.
```

Reply:
I hadn't intended that this would be part of "Scheduling Constraints", but I can see how it might be confusing where it is.  I'll see if it makes sense elsewhere.

Reply:
I moved this and took your wording.  See if it makes sense now.